frontend/src/components/ModelInfo.tsx

```diff
@@ -14,27 +14,12 @@
  * limitations under the License.
  */
 import * as React from 'react';
-import {stylesheet, classes} from 'typestyle';
-import {color, commonCss} from '../Css';
-import {formatDateString} from '../lib/Utils';
-
-const css = stylesheet({
-  modelInfo: {
-    display: 'flex',
-    flexDirection: 'row',
-    flexWrap: 'wrap',
-  },
-  field: {
-    flexBasis: '300px',
-    marginBottom: '20px',
-  },
-  term: {
-    color: color.grey,
-  },
-  value: {
-    color: color.secondaryText,
-  }
-});
+import {classes} from 'typestyle';
+import {commonCss} from '../Css';
+import {getArtifactProperty, logger} from '../lib/Utils';
+import {css} from './ArtifactInfo';
```

Comment:
This should probably be put in a more common location instead of pulled from a different component

Reply:
Yeah I wasn't quite sure what the best pattern for doing that would be since the ModelInfo was just adding terms to its parent. Given your comment above, maybe the best thing to do is just make a functional component that for the wrapper div, dt, and dd elements. For the sake of time (and the fact that this has other changes coupled with it), I'd like to push that off to a subsequent change but I'll make a TODO for it if you'd like.